// Exceptions/ToIntException.kt

fun erroneousCode() {
  // Uncomment this line to get an exception:
  // println("1$".toInt())    // [1]
}

fun main(args: Array<String>) {
  erroneousCode()
}
